src/modules/ROOT/pages/changelog/202303/01.json

```diff
@@ -0,0 +1,889 @@
+        "https://github.com/cortezaproject/corteza/pull/670/commits/a4308e8d88cdda2f13e3f19231b7815d6812d79e"
+      ]
+    },
+    {
+      "what": [
+        "support for managing and generating user's profile picture."
+      ],
+      "why": [
+        "to provide users with greater flexibility and customization options for their profile avatars, while also allowing admins to manage avatars on behalf of users."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/887/commits/e24b391e5b1dc539855122534a4394a6868c4afd"
+      ]
+    },
+    {
+      "what": [
+        "support for record list filters.",
+        "Record list filters can be configured on the record list configurator.",
+        "They can be grouped by roles and can be quickly selected from the filter dropdowns."
+      ],
+      "why": [
+        "to allow users to quickly apply filter to the record list."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/921/commits/63f1647f7ba545f4811b6d20a960d14c095aa52b",
+        "https://github.com/cortezaproject/corteza/pull/921/commits/1815e1daa8a4d52bf57d4bb1ebdbdbdbc1587344",
+        "https://github.com/cortezaproject/corteza/pull/921/commits/24cbb818b7e944578692ca59e9d61f1f6e711c10"
+      ]
+    },
+    {
+      "what": [
+        "support for inline editing in record lists."
+      ],
+      "why": [
+        "to allow user to easily edit a record by just hovering on the field and pressing on the edit button."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/956/commits/363afde876a22b454dc324b870902c5a9f70bb84"
+      ]
+    },
+    {
+      "what": [
+        "support for geo location capture.",
+        "Ability to set current location while on a map was also added.",
+        "An option to prefill the current location and to hide the current location button was added."
+      ],
+      "why": [
+        "to improve the user experience and the location accuracy of the users.",
+        "Now instead of manually entering their coordinates, they can just use the `find me` button and the coordinates will be updated."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/955/commits/1628845eccbbcaf8f939efdd1241e7dd147da8b4",
+        "https://github.com/cortezaproject/corteza/pull/955/commits/de0f2ef3d103256a0ddff0fe03acd892e4ebce2f",
+        "https://github.com/cortezaproject/corteza/pull/955/commits/6a01e35f06b848867a14b35b6e7649c0e358ba06",
+        "https://github.com/cortezaproject/corteza/pull/955/commits/2e45899fad36d9495f8ff8625eb4929230e53604"
+      ]
+    },
+    {
+      "what": [
+        "support for page icon and the option to expand sub-pages in the sidebar.",
+        "The expand sub-page option was added to improve access to sub-pages.",
+        "When the option is enabled, the user can always see sub-pages without having to constantly expand the parent page."
```

Comment:
Should the page icon be its own entry?
I'd consider splitting this into two, or moving the expand sub-pages into the generic UI/UX change entry